#!/usr/bin/python3


def raise_exception_msg(message=""):
    """Raise a NameError exception with personalized message"""
    raise NameError(message)
